src/scripts/extension-utils.js

```diff
@@ -72,10 +70,11 @@ const sendAsyncMessageToContentScript = async payload => {
   return response
 }
 
-export const getAccounts = () => {
-  sendMessageToContentScript({
+export const getWalletsFromExtension = async () => {
```

Comment:
We agreed to name `wallets` `accounts` in this context as we only send the name and the address not the actual keys.

Reply:
Good thinking.